src/init/inst-functions.sh

```diff
@@ -904,6 +904,7 @@ InstallCommon()
         if [ -f  ../etc/ossec.mc ]; then
             ${INSTALL} -m 0660 -o root -g ${WAZUH_GROUP} ../etc/ossec.mc ${INSTALLDIR}/etc/ossec.conf
         else
+            echo "ERROR: unable to generate ossec.conf file with desired configurations, using default configurations from ${OSSEC_CONF_SRC}"
```

Comment:
Question: Isn't it better to use a **WARNIG** message?
```suggestion
            echo "WARNING: unable to generate ossec.conf file with desired configurations, using default configurations from ${OSSEC_CONF_SRC}"
```

Reply:
I think you are right! I will change it